01_06_Leveling_Up.txt

```diff
@@ -15,48 +15,48 @@ skill points for that level.
-| 5     | level    |       | 1          | +1         |               | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 6     | "        | 4     |            |            | 5             | 3d10               |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 7     | "        |       |            |            |               | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 8     | "        | 5     |            |            | 7             | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 9     | "        |       |            | +1         |               | 4d10               |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 10    | "        | 6     | 2          |            | 8             | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 11    | "        |       |            |            |               | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 12    | "        | 7     |            |            | 10            | 5d10               |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 13    | "        |       |            |            |               | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 14    | "        | 8     |            |            | 11            | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 15    | "        |       | 3          | +1         |               | 6d10               |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 16    | "        | 9     |            |            | 13            | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 17    | "        |       |            |            |               | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 18    | "        | 10    |            |            | 14            | 7d10               |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 19    | "        |       |            |            |               | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 20    | "        | 11    | 4          | +1         | 16            | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
```

Comment:
That's a reward that is going to feel very good! It's worth calling out that this is going to have major balance ramifications on enemy levels, but I think this is ready to test. Looking forward to it!